daemon/container.go

```diff
@@ -253,6 +253,7 @@ func (container *Container) Start() (err error) {
 			container.toDisk()
 			container.cleanup()
 			container.LogEvent("die")
+			container.LogEvent("exit")
```

Comment:
Let me elaborate, I think its important to has consistent "book-ends" around the events for a container: 'create' at the start, and 'die' at the end.  This allows for a consistent processing of the events stream - the user can be assured that once they see 'die' they can stop looking for events for that container.  With this code they now have to wait (for a racey amount of time) to see if 'exit' shows up later on.  It makes it much more complicated w.r.t. the event processing.